common/math/math.go

```diff
@@ -72,38 +70,38 @@ func CalculateInformationRatio(values, benchmarkRates []float64, isGeometric boo
 			benchmarkRates = append(benchmarkRates, benchmarkRates[0])
 		}
 	}
-	avgValue := CalculateTheAverage(values, isGeometric)
-	avgComparison := CalculateTheAverage(benchmarkRates, isGeometric)
 	var diffs []float64
 	for i := range values {
 		diffs = append(diffs, values[i]-benchmarkRates[i])
 	}
-	stdDev := CalculatePopulationStandardDeviation(diffs)
+	stdDev := PopulationStandardDeviation(diffs)
 	if stdDev == 0 {
 		return 0
 	}
-	return (avgValue - avgComparison) / stdDev
+	return (averageValues - averageComparison) / stdDev
 }
 
-// CalculatePopulationStandardDeviation calculates standard deviation using population based calculation
-func CalculatePopulationStandardDeviation(values []float64) float64 {
+// PopulationStandardDeviation calculates standard deviation using population based calculation
+func PopulationStandardDeviation(values []float64) float64 {
 	if len(values) == 0 {
```

Comment:
if len(values) < 2

Reply:
I don't understand. Its to ensure that we don't divide by zero